lib/sles4sap/microsoft_sdaf_basetest.pm

```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+#
+# Basetest used for Microsoft SDAF deployment
+
+package sles4sap::microsoft_sdaf_basetest;
+use strict;
+use warnings;
+use testapi;
+use parent 'opensusebasetest';
+use sles4sap::microsoft_sdaf;
+
+sub post_fail_hook {
```

Comment:
Doesn't this mean that the post_fail_hook will do nothing?

Reply:
Yes.. it's just a placeholder for SDAF test modules. I already have the code it will just be added in future PR's. It will include cleanup for SDAF deployment and stuff like that.